tsdb/engine/tsm1/engine_test.go

```diff
@@ -16,6 +19,7 @@ import (
 	"github.com/influxdata/influxdb/pkg/deep"
 	"github.com/influxdata/influxdb/tsdb"
 	"github.com/influxdata/influxdb/tsdb/engine/tsm1"
+	// "github.com/pkg/profile"
```

Comment:
I like that better since it won't potentially get overwritten by something like `goimports`.